src/test/scala/ch/epfl/bluebrain/nexus/kg/resources/AdditionalValidationSpec.scala

```diff
@@ -260,6 +266,25 @@ class AdditionalValidationSpec
         validation(id, schema, types, resource.value, 1L).value.accepted shouldEqual resource.value
       }
     }
+
+    "applied to storages" should {
```

Comment:
add additional test cases for:
- when the volume is not writable
- when the volume is not a directory